schemas/TransferTemplate.json

```diff
@@ -20,6 +20,11 @@
       "description": "The current state of the transfer (informational only)",
       "$ref": "TransferState.json"
     },
+    "rejected_reason": {
```

Comment:
We already use "cancelled"/"canellation", "executed"/"execution"; should we call it "rejection_reason"? Currently it reads as a "reason that was rejected".